google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -53,9 +53,6 @@ def set_run(self, run: str):
         )
         self._run = execution.name
 
-    def log_param(self, name: str, value: Union[float, int, str]):
```

Comment:
you will have to update a lot of the ValueError descriptions in this file. 

For example:

"No experiment set for this run. Make sure to call aiplatform.init(experiment='my-experiment') before trying to set_run. "

Same with others. Check whether it will be set_run or start_run